lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/DriverStatusManager.java

```diff
@@ -180,17 +180,17 @@ public synchronized void sendJobEndingMessageToClient(final Optional<Throwable>
     if (this.driverTerminationHasBeenCommunicatedToClient) {
       LOG.log(Level.SEVERE, ".sendJobEndingMessageToClient() called twice. Ignoring the second call");
     } else {
-      { // Log the shutdown situation
-        if (this.shutdownCause.isPresent()) {
-          LOG.log(Level.WARNING, "Sending message about an unclean driver shutdown.", this.shutdownCause.get());
-        }
-        if (exception.isPresent()) {
-          LOG.log(Level.WARNING, "There was an exception during clock.close().", exception.get());
-        }
-        if (this.shutdownCause.isPresent() && exception.isPresent()) {
-          LOG.log(Level.WARNING, "The driver is shutdown because of an exception (see above) and there was an exception during clock.close(). Only the first exception will be sent to the client");
-        }
+      // Log the shutdown situation
```

Comment:
This seems like a pure formatting change, right? If so, please remove it from the PR.